README.md

```diff
@@ -30,6 +30,62 @@ Please raise a new pull request for new library additions and existing library u
 
 ## Adding a new or updating an existing library
 
+There is a simple way to add a library via single package.json only.
+
+You can follow these steps:
+
+1. Fork cdnjs repository to your GitHub account.
+
+2. Create a new folder named **libname** under `/ajax/libs`, then create a package.json under `/libname`.
+
+    > A tip: you can type `/libname/package.json` after cdnjs/ajax/libs/ to create them if you use web interface
+
+    > Note: the libname must match the name in package.json
+
+3. Edit package.json:
+
+    take the npm package lib `frzr` for an example:
+    ```js
+    {
+      "name": "frzr",
+      "description": "Turboboosted 2 KB view library",
+      "homepage": "https://frzr.js.org",
+      "repository": {
+        "type": "git",
+        "url": "git://github.com/pakastin/frzr"
+      },
+      "author": "Juha Lindstedt",
+      "license": "ISC",
+      "keywords": [
+        "frzr",
+        "tiny",
+        "view"
+      ],
+      "filename": "frzr.min.js",
+      "npmName": "frzr",
+      "npmFileMap": [
+        {
+          "basePath": "dist",
+          "files": [
+            "frzr*.js"
+          ]
+        }
+      ]
+    }
+    ```
+    you can change to use git auto-update, [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md#enabling-npmrecommended-or-git-auto-update)
+
+    > Note:
+    Don't add version in package.json if you don't add it with real file.
```

Comment:
`if you don't add it with real file` -> `if you don't want to commit with its assets`